NDB.Covid19/NDB.Covid19/Locales/nb.json

```diff
@@ -25,12 +25,12 @@
   "WELCOME_PAGE_THREE_BODY_ONE": "Mobilen din lager en tilfeldig ID-nøkkel hvert 15. minutt. Når to mobiler er i nærheten av hverandre, sender de disse nøklene til hverandre. ",
   "WELCOME_PAGE_THREE_BODY_TWO": "Du kan dele informasjonen med andre om du får covid-19. Da får andre som har vært i nærheten av deg beskjed i appen.",
   "WELCOME_PAGE_THREE_INFOBOX_BODY": "Appen bruker Bluetooth. Teknologien bak appen er utviklet av Apple og Google. ",
-  "WELCOME_PAGE_ACCESSIBILITY_ANNOUNCEMENT_PAGE_CHANGED_TO_THREE": "Side 3 av 4",
+  "WELCOME_PAGE_ACCESSIBILITY_ANNOUNCEMENT_PAGE_CHANGED_TO_THREE": "Side 4 av 4",
   "WELCOME_PAGE_FOUR_TITLE": "Appen passer på personopplysningene dine",
   "WELCOME_PAGE_FOUR_BODY_ONE": "Du kan ikke se hvem eller hvor mange du har vært i nærheten av, og de kan ikke se deg.",
   "WELCOME_PAGE_FOUR_BODY_TWO": "Hvis du melder deg smittet, kan ikke andre se at det er du som gjør det.",
   "WELCOME_PAGE_FOUR_BODY_THREE": "Appen registrerer ikke data om hvor du er. ",
-  "WELCOME_PAGE_ACCESSIBILITY_ANNOUNCEMENT_PAGE_CHANGED_TO_FOUR": "Side 4 av 4",
+  "WELCOME_PAGE_ACCESSIBILITY_ANNOUNCEMENT_PAGE_CHANGED_TO_FOUR": "Side 3 av 4",
```

Comment:
There's something off about these. Either the keys or values should be swapped

Reply:
nope - these are good for now. We have included another PR to handle this.

Reply:
they are actually swapped by presenter, yes it should be fixed more properly, but for now this will work.